docs/usage/notes.rst

```diff
@@ -233,10 +236,19 @@ won't free disk space, they merely tag data as deleted in a new transaction.
 Be aware that as soon as you write to the repo in non-append-only mode (e.g. prune,
 delete or create archives from an admin machine), it will remove the deleted objects
 permanently (including the ones that were already marked as deleted, but not removed,
-in append-only mode).
+in append-only mode). Automated edits to the repository (such as a cron job running
+``borg prune``) will render append-only mode moot if data is deleted.
```

Comment:
Is this about running `borg prune` with or without append-only?
"if data is deleted" by that automated `borg prune` (without append-only) or "marked" as deleted (with append-only) before that automated `borg prune` (without append-only)?
I read this dozens of times, and I still struggle to understand this sentence and what it tries to suggest.

Reply:
No, this is about `compact_segments`, which borg < 1.2 implicitly calls at the end of repo-writing commands to get rid of most unused [or rather "not used any more"] parts of segment files IF the repo is not append-only.

In append-only mode, it just keeps everything it ever had, never frees any space, always appends changes to the end.

Reply:
And the sentence tries to suggest that you should make sure that your repo still shows all you want to keep and that all archives you want to keep actually have all content you want to keep.

If an attacker used borg delete while the repo is append-only, there is no immediate damage - you could still rollback the repo to any past state. But if you switch off a-o and write, compact_segments will actually get rid of the deleted data and no rollback is possible any more.